tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -388,3 +388,217 @@ The following tools may be used for performance and memory profiling of your man
 - Visual Studio.
 
 Profiling managed and unmanaged code at once is possible with both JetBrains tools and Visual Studio, but limited to Windows.
+
+Using ``async``/``await``
+-----------------
+
+You might face a scenario where you must ``await`` a method call.
+You will notice that when you use ``await``, you are required to mark the method you use it in as ``async``, and change the return type to ``Task`` or ``Task<T>``. 
```

Comment:
Probably or `ValueTask`/`ValueTask<T>` (or any generalized async type).

Reply:
I can reword this "change the return type to any ``await``-able return type, such as ``Task`` or ``Task<T>``". Sound good?